cpp/src/grpc/server/grpc_server_main.cpp

```
@@ -0,0 +1,281 @@
+    return 1;
+  }
+  shm_ctrl = static_cast<SharedMemoryControl*>(
+    mmap(nullptr, sizeof(SharedMemoryControl), PROT_READ | PROT_WRITE, MAP_SHARED, ctrl_shm_fd, 0));
+  close(ctrl_shm_fd);
+  if (shm_ctrl == MAP_FAILED) {
+    std::cerr << "[Server] Failed to mmap control: " << strerror(errno) << "\n";
+    return 1;
+  }
+
+  for (size_t i = 0; i < MAX_JOBS; ++i) {
+    memset(&job_queue[i], 0, sizeof(JobQueueEntry));
```

Cppcheck (2.19.0)

[error] Using 'memset' on struct that contains a 'std

(memsetClass)